Core/Core/Planner/APICalendarEvent.swift

```diff
@@ -155,8 +157,25 @@ public struct GetCalendarEventsRequest: APIRequestable {
         if let allEvents = allEvents {
             query.append(.bool("all_events", allEvents))
         }
+
         return query
     }
+
+    private func createDateString(from date: Date?) -> String {
+        if let date = date {
+            return dateFormatter().string(from: date)
+        } else {
+            return ""
+        }
+    }
+
+    public func dateFormatter() -> DateFormatter {
```

Comment:
Creating date formatters used to be slow, we could move this to a lazy variable since we have to use this at least two times.

Reply:
Moved it to a private static constant.